TODO.md

```diff
@@ -10,3 +10,8 @@
     <ion-menu-button menu="menuAppId"></ion-menu-button>
   </ion-button>
 ```
+
+
+## Christopher notes
```

Comment:
please can you remove this useless comment ?

Reply:
Done !